src/osd/osd_types.cc

```diff
@@ -276,32 +276,35 @@ void request_redirect_t::generate_test_instances(list<request_redirect_t*>& o)
 
 void objectstore_perf_stat_t::dump(Formatter *f) const
 {
-  f->dump_unsigned("commit_latency_ms", os_commit_latency);
-  f->dump_unsigned("apply_latency_ms", os_apply_latency);
+  // *_ms values just for compatibility.
+  f->dump_float("commit_latency_ms", os_commit_latency_ns / 1000000.0);
```

Comment:
note, unsigned -> float here

Reply:
Presumably the motivation for having commit_latency_ms in addition to commit_latency_ns is to have backward compatibility, so it probably doesn't make sense to change the type of the _ms field.

Reply:
But JSON (as a format) internally does not distinguish between `int` and `float`. So after my patch any correct application that reads JSON should see enhancement.